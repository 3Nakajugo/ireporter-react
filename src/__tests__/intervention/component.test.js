import React from 'react';
import { shallow } from 'enzyme';
import InterventionComponent from '../../components/Intervention'


describe('<InterventionComponent />', () => {
  const props = {
    state: {
      interventions:[ {
        comment: "floods",
        createdby: "consolatem",
        date: "Fri, 28 Jun 2019 04:35:28 GMT",
        file: "edna.jpg",
        incident_id: 1,
        incident_type: "intervention",
        location: "ntinda",
        status: "draft"
      }]
    }
  };
  it('should render intervention component', () => {
    const wrapper = shallow(<InterventionComponent {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});