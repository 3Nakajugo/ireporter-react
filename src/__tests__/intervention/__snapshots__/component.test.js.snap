// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InterventionComponent /> should render intervention component 1`] = `ShallowWrapper {}`;
